src/main/java/com/ciandt/techgallery/service/impl/TechnologyFollowersServiceImpl.java

```diff
@@ -0,0 +1,137 @@
+package com.ciandt.techgallery.service.impl;
+
+import com.google.api.server.spi.response.BadRequestException;
+import com.google.api.server.spi.response.InternalServerErrorException;
+import com.google.api.server.spi.response.NotFoundException;
+
+import com.googlecode.objectify.Ref;
+
+import com.ciandt.techgallery.persistence.dao.TechnologyFollowersDAO;
+import com.ciandt.techgallery.persistence.dao.impl.TechnologyFollowersDAOImpl;
+import com.ciandt.techgallery.persistence.model.TechGalleryUser;
+import com.ciandt.techgallery.persistence.model.Technology;
+import com.ciandt.techgallery.persistence.model.TechnologyFollowers;
+import com.ciandt.techgallery.service.TechnologyFollowersService;
+import com.ciandt.techgallery.service.TechnologyService;
+import com.ciandt.techgallery.service.UserServiceTG;
+import com.ciandt.techgallery.service.enums.ValidationMessageEnums;
+import com.ciant.techgallery.transaction.Transactional;
+
+import java.util.ArrayList;
+
+/**
+ * Services for Technology Endpoint requests.
+ *
+ * @author felipers
+ *
+ */
+public class TechnologyFollowersServiceImpl implements TechnologyFollowersService {
```

Comment:
@felipeaziz :  I forgot to tell you that you must add @Transactional annotation on the class declaration as well.